src/script/mod.rs

```diff
@@ -178,100 +298,65 @@ impl Code<'_> {
     }
 
     /// Produce an [`Opcode`] iterator from [`Code`].
+    ///
+    /// It returns `Err` with the length if the script is too long.
     pub fn parse(&self) -> Parser<'_> {
```

Comment:
What is `It` here? The method doesn't return a `Result`; are you saying that the returned `Parser` will contain a single `Err` item in this case?